packages/webviz-core/src/panels/ThreeDimensionalViz/SceneBuilder/index.js

```diff
@@ -796,6 +798,25 @@ export default class SceneBuilder implements MarkerProvider {
         // flatten btn: set empty z values to be at the same level as the flattenedZHeightPose
         this._consumeOccupancyGrid(topic, deepParse(message));
         break;
+      case NAV_MSGS_PATH_DATATYPE: {
+        const pathStamped = cast<BinaryPath>(message);
+        if (pathStamped.poses().length() === 0) {
+          break;
+        }
+        const newMessage = {
+          header: deepParse(pathStamped.header()),
+          // TODO(@cjds) Make this make use of the orientation of the poses in the path as well
```

Comment:
Curious what do you have in mind for orientation visualization? Some sort of arrow?